Makefile

```diff
@@ -68,3 +68,11 @@ assisted_deployment:
 
 assisted_deployment_requirements:
 	./assisted_deployment.sh install_prerequisites_assisted_service
+
+fleeting: fleeting_configure requirements configure fleeting_deploy_iso bell
```

Comment:
I don't have a strong opinion on the name, but would it be more intuitive to call this `agent_deployment` or something?

Also the order looks inconsistent with our existing sequence, should probably be `requirements configure fleeting_configure fleeting_deploy_iso bell` - that way you can always depend on the prerequisites being installed (like ansible for example)

Reply:
I originally did call it agent. Since we also have assisted-installer-agent I thought that might be confusing so I went with fleeting instead. I don't have a strong opinion either. We are still trying to figure out what to call ourselves. 🤣